containers/datalab/content/run.sh

```diff
@@ -56,6 +56,13 @@ if [[ -n "${GATEWAY_VM}" ]]; then
   fi
 fi
 
+if [[ "${CLI_LOGIN}" == "true" ]]; then
+  USER_EMAIL=`gcloud auth list --format="value(account)"`
+  if [[ -z "${USER_EMAIL}" ]]; then
+    gcloud auth login
+  fi
+fi
```

Comment:
Also, if auth failed it will exit gracefully with right message? Just double check.